src/pages/ManageValidator/LeftManage/ValidatorBlueSection/ValidatorBalance.tsx

```diff
@@ -29,8 +32,8 @@ const ValidatorBalance = () => {
           $0 USD
         </Text>
       </YStack>
-      <Stack marginRight="5rem">
-        <Icon src="/icons/suit-guy.png" width={200} height={70} />
+      <Stack marginRight="3rem">
+        <Icon src="/icons/suit-guy.png" width={200} height={70}></Icon>
```

Comment:
Ditto. This was changed [here](https://github.com/nimbus-gui/nimbus-gui/commit/6a92e4e0b6709162d05969d85faaefad7a293af1#diff-31ee1c410505317957ce3bb2ecd8716bf266b54b65a4149f33f32779a6fe7c0e)